torch/csrc/cuda/Graph.cpp

```diff
@@ -35,7 +35,7 @@ void THCPGraph_init(PyObject *module) {
       .def("capture_begin",
            &::at::cuda::CUDAGraph::capture_begin,
            py::call_guard<py::gil_scoped_release>(),
-           py::arg("pool") = c10::cuda::MempoolId_t{0, 0})
+           py::arg("pool") = c10::hip::MempoolId_t{0, 0})
```

Comment:
Any reason to change this manually instead of hipify?

Reply:
Fixed, it had already hipified before I committed